Makefile

```diff
@@ -46,10 +46,11 @@ dev_db:
 	cd server/scripts/create_db_wrapper && ./create_db_wrapper.sh
 
 # Install all deps for this project.
+# Note: PNPM must be installed separately for each version of NPM used, since it is installed within each NPM installation.
 deps:
-	. $(NVM_SCRIPT) && nvm install 22 && pnpm i
-	. $(NVM_SCRIPT) && cd frontend && nvm install 16 && npm i --legacy-peer-deps
-	. $(NVM_SCRIPT) && cd frontend-v2 && nvm install 18 && npm i -g pnpm && pnpm i
+	. $(NVM_SCRIPT) && nvm i 22 && npm i -g pnpm && pnpm i
+	. $(NVM_SCRIPT) && cd frontend && nvm i 16 && npm i --legacy-peer-deps
+	. $(NVM_SCRIPT) && cd frontend-v2 && nvm i 18 && npm i -g pnpm && pnpm i
```

Comment:
Random question... but do you know why we don't use node 22 for the new frontend v2 app?

Reply:
(Not blocking)

Reply:
I don't think there's any hard reason. Your scaffold react app [commit](https://github.com/dxe/adb/commit/f02c797c0224d0076e84ebc2ebe6a4d68c2ef104#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R9) introduced 18. Then I added 22 for linting/formatting purposes and didn't bother trying to upgrade the React one yet since we already had it set up to switch between versions easily and I didn't wanna change multiple things at once.

Reply:
interesting, not sure why it did 18.